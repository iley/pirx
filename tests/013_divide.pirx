func main(): int {
  var x: int;
  var y: int;
  x = 12;
  y = 3;
  printf("x / y = %d\n", x / y);
  printf("x / 2 = %d\n", x / 2);
  printf("x + y / 3 = %d\n", x + y / 3);
  return 0;
} 
