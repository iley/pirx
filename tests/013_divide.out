x / y = 4
x / 2 = 6
x + y / 3 = 13
